### DISCUSSION TITLE
Client API Documentation

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#6342] [Q&A] [answered]
Does anyone know how to export or see the Mirth Client API documentation?  I can access it via the API itself, but when I click on the following, it just errors out.   I see there is an open bug issue submitted about it last year, with no update:

https://localhost:8443/api/openapi.json

https://github.com/nextgenhealthcare/connect/issues/5896

**Comment by pacmano1** [comment, marked as answer]
To download that JSON:

```
curl -u yourusername:yourpasword -H "X-Requested-With:OpenAPI" https://<mirthServerIP>:8443/api/openapi.json --insecure
```
